agent/ecscni/plugin.go

```diff
@@ -180,11 +189,18 @@ func (client *cniClient) cleanupNS(cfg *Config) error {
 		return errors.Wrap(err, "cni cleanup: invoke bridge plugin failed")
 	}
 	seelog.Debugf("[ECSCNI] bridge cleanup done: %s", cfg.ContainerID)
-
+	// clean up eni network namespace
 	err = client.del(runtimeConfig, cfg, client.createENINetworkConfig)
 	if err != nil {
 		return errors.Wrap(err, "cni cleanup: invoke eni plugin failed")
 	}
+	if cfg.AppMeshCNIEnabled {
+		// clean up app mesh network namespace
```

Comment:
Could you add a debug log for AppMesh so we can know whether it is called or not?

Reply:
added